### ISSUE TITLE
Add an option to pick a version while installing on Android

### ISSUE THREAD

**Issue by nozwock** [nozwock/yanu#8]
The current installation command is only capable of installing the `latest` version.
```sh
bash <(curl -L https://raw.githubusercontent.com/nozwock/yanu/main/resources/scripts/yanu-bootstrap-termux_proot.sh)
```
Sometimes it's required to be able to choose a specific version in case of issues or for debugging, etc.

Add an argument in the bootstrap script to accept an optional version number, and then continue to install the given version instead of the `latest`, for eg-
```sh
bash <(curl -L https://raw.githubusercontent.com/nozwock/yanu/main/resources/scripts/yanu-bootstrap-termux_proot.sh) -- -v 0.6.0
```
Passing `-- -v 0.6.0` to install the `0.6.0` version instead of `latest`.

Looks like `latest/download` would need to be replaced with `download/{version_tag}` in [this line](https://github.com/nozwock/yanu/blob/553a0172d880caec7ede063c83f22687adb7c597/resources/scripts/yanu-bootstrap-termux_proot.sh#L25) to download the bin of a specific version.

**Comment by nekomekoraiyuu** [comment]
Its possible to do this
but everytime yanu gets a new release you would have to add the new version to the array list (Lazy implementation)
i mean theres a better way to do this like make modification to build script then add vers to another file (some bash/rust magic)

**Comment by nozwock** [comment]
> but everytime yanu gets a new release you would have to add the new version to the array list (Lazy implementation)

I don't think so, just pretend that the version input given by the user is correct and substitute the `/latest/download` with `/download/$1`

I don't how this would be hard?

**Comment by nekomekoraiyuu** [comment]
> > but everytime yanu gets a new release you would have to add the new version to the array list (Lazy implementation)
> 
> I don't think so, just pretend that the version input given by the user is correct and substitute the `/latest/download` with `/download/$1`
> 
> I don't how this would be hard?

well nevermind its not that hard 
i found a solution to that
it'll list available release tags now
(i will do some sed/grep magic)
xD

**Comment by nekomekoraiyuu** [comment]
doing the implementation in my own style (user friendly) :remu: 👍

**Comment by nekomekoraiyuu** [comment]
#9 should do

**Comment by nozwock** [comment]
Closed by #9
